### DISCUSSION TITLE
Nominate Tyler Graff and Zach Jones to Nebari Triage Team

### DISCUSSION THREAD

**Discussion by kenafoster** [nebari-dev/nebari#2619] [Community]
I nominate Tyler Graff @tylergraff and Zach Jones @joneszc to Nebari Triage Team

Both have engagement that qualifies per [Nebari Team Structure.](https://www.nebari.dev/docs/community/team-structure/#join-a-team)

Tyler has two PRs merged: https://github.com/nebari-dev/nebari/pulls?q=is%3Apr+author%3Atylergraff+is%3Amerged+
Zach has two currently open PRs: https://github.com/nebari-dev/nebari/pulls?q=is%3Apr+author%3Ajoneszc

**Comment by kenafoster** [comment]
@dharhas or @dcmcand what do you think?

**Comment by dharhas** [comment]
Approved. Invitations sent. We can close this discussion once they accept.

**Reply by tylergraff** [reply]
Accepted!
